### PR TITLE
chore(ci): Add trigger to update Paneron website after a release has been pushed

### LINKED ISSUE #53: Trigger re-deploy action in paneron.org after version changes
We should trigger re-deploy workflow action in paneron.org after bumping version.

### PR DESCRIPTION
**Pull request by ribose-jeffreylau**
Fixes #53

### CODE REVIEW

File: .github/workflows/update-website.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  trigger-website-build:
+    name: Trigger website build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Dispatch update-website event
```
Comment (ronaldtse) [inline .github/workflows/update-website.yml:13]: Can we reuse actions like this?

* https://github.com/marketplace/actions/workflow-dispatch
* https://github.com/marketplace/actions/dispatch-action

Comment (ribose-jeffreylau) [reply .github/workflows/update-website.yml:13]: Technically we could, but I feel that the Curl interface is more well known.

Note that GitHub has a native way to use workflow-dispatch via "uses:", so [the workflow-dispatch action](https://github.com/marketplace/actions/workflow-dispatch) may not add much value here, if at all.

Comment (ronaldtse) [reply .github/workflows/update-website.yml:13]: What's the reason we're not using the "uses:" function? The curl syntax is rather obscure...

Comment (ronaldtse) [reply .github/workflows/update-website.yml:13]: Discussed in person, let's go!

**Pull request merged**